throughput/spam.go

```diff
@@ -143,19 +145,21 @@ func (s *Spammer) Spam(ctx context.Context, sh SpamHelper, terminate bool, symbo
 		issuer.Start(cctx)
 	}
 
-	// set logging
-	s.tracker.logState(cctx, issuers[0].cli)
+	// start logging
+	s.tracker.startPeriodicLog(ctx, cli)
```

Comment:
Do we still need `cctx` created above if we're not longer using it here to stop the tracker?